lib/mutant/subject.rb

```diff
@@ -9,11 +9,10 @@ class Subject
     # @return [Enumerable<Mutation>]
     # @return [undefined]
     def mutations
-      mutations = [neutral_mutation]
-      Mutator.each(node) do |mutant|
-        mutations << Mutation::Evil.new(self, wrap_node(mutant))
+      [neutral_mutation] +
+      Mutator.each(node).map do |mutant|
```

Comment:
I don't think there's any strong reason to have this on it's own line. In fact. you can probably use `Array#concat` to save a tiny bit of memory allocation.